packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -108,6 +109,7 @@ export class ClientMemoryState {
  * active in multiple tabs.
  */
 class CachedTargetIdGenerator {
+  // TODO(wuandy): rename this to targetMapping.
   private queryMapping: QueryMap = {};
```

Comment:
Should we make this an ObjectMap<Target, number>? We should then be able to remove most calls to `canonicalId()`.

Reply:
Good idea, done.